doc-content/shared-kie-docs/src/main/asciidoc/KieServer/kie-server-commands-samples-ref.adoc

```diff
@@ -0,0 +1,876 @@
+                    "java.util.Date": 1538504619749
+                  },
+                  "content": [
+                    "Container command-script-container successfully created with module com.redhat:Project1:1.0."
+                  ]
+                }
+              ],
+              "container-alias": null
+            }
+          ]
+        }
+      }
+    }
+  ]
+}
+----
+--
+CallContainerCommand::
++
+--
+Calls a KIE container and executes one or more runtime commands. For information about {PRODUCT} runtime commands, see xref:runtime-commands-con_kie-apis[].
+
+.Command attributes
+[cols="30%,50%,20%", frame="all", options="header"]
+|===
+|Name
+|Description
+|Requirement
+
+|`containerId`
+|ID of the KIE container to be called
+|Required
+
+|`payload`
+|One or more commands in a `BatchExecutionCommand` wrapper to be executed on the KIE container
+|Required
+
+|
+|
+|Optional
+|===
+
+*<@Maciej/@Tiho: The above attributes table is missing the rest of the required and optional parameters, and the below example Java command contains only the required payload portion and needs the rest of the call-container command to be complete.>*
+
+.Example Java command
+[source,java]
+----
+import org.kie.server.api.marshalling.Marshaller;
+import org.kie.server.api.marshalling.MarshallerFactory;
+import org.kie.server.api.marshalling.MarshallingFormat;
+
+...
+
+Marshaller marshaller = MarshallerFactory.getMarshaller(MarshallingFormat.JSON,myclass.class.getClassLoader());
+//Marshalling format is changed based on the method of marshalling for the CallContainerCommand. Also note myclass.class classloader is called. If replicating this code, change the name to the name of your class.
+
+Command<?> fire = KieServices.Factory.get().getCommands().newFireAllRules();
+BatchExecutionCommand batch = KieServices.Factory.get().getCommands().newBatchExecution(Arrays.<Command<?>>asList(fire), "defaultKieSession");
+String payload = marshaller.marshall(batch);
+----
+
+.Example REST request body (JSON)
+[source,json]
+----
+{
+  "commands" : [ {
+    "call-container" : {
+      "payload" : "{\n  \"lookup\" : \"defaultKieSession\",\n  \"commands\" : [ {\n    \"fire-all-rules\" : {\n      \"max\" : -1,\n      \"out-identifier\" : null\n    }\n  } ]\n}",
+      "container-id" : "command-script-container"
+    }
+  } ]
+}
+----
+
+.Example server response (JSON)
+[source,json]
+----
+{
+  "response": [
+    {
+      "type": "SUCCESS",
+      "msg": "Container employee-rostering successfully called.",
+      "result": "{\n  \"results\" : [ ],\n  \"facts\" : [ ]\n}"
+    }
+  ]
+}
+----
+--
+DisposeContainerCommand::
++
+--
+Disposes a specified KIE container in the {KIE_SERVER}.
+
+.Command attributes
+[cols="30%,50%,20%", frame="all", options="header"]
+|===
+|Name
+|Description
+|Requirement
+
+|`containerId`
+|ID of the KIE container to be disposed
+|Required
+
+|
+|
+|Optional
+|===
+
+.Example Java command
+[source,java]
+----
+
```

Comment:
do you want something like this 

````
client.disposeContainer(CONTAINER_ID_101);
````

that is not really a command call but client api call that underneath uses the command.

Reply:
Right. Done.